images/fcos-bastion-image/Containerfile

```diff
@@ -3,4 +3,4 @@ FROM quay.io/openshifttest/fcos-bm:base
 COPY root/ /
 
 RUN set -x; pip3 install --target=/usr/lib redfish pandas \
-    && systemctl preset-all
+    && pip3 install netaddr junos-eznc wheel && systemctl preset-all
```

Comment:
Hi Jad, add the three deps one line before together with redfish and pandas. Also, let's ensure the build (`images` job) succeeds.